packages/connect/src/api/firmware/getBinary.ts

```diff
@@ -1,26 +1,22 @@
-import type { FirmwareRelease } from '@trezor/device-utils';
-
 import { httpRequest } from '../../utils/assets';
 
 const ALL_SLASHES_AT_THE_END_REGEX = /\/+$/;
 
-interface GetBinaryProps {
+interface GetBinaryParams {
     baseUrl: string;
-    btcOnly?: boolean;
-    release: FirmwareRelease;
+    firmwareName: string;
 }
 
-export const getBinary = ({ baseUrl, btcOnly, release }: GetBinaryProps) => {
-    const fwUrl = release[btcOnly ? 'url_bitcoinonly' : 'url'];
+export const getBinary = ({ baseUrl, firmwareName }: GetBinaryParams) => {
```

Comment:
this sanity check https://github.com/trezor/trezor-suite/blob/feat/firmware-update/packages/connect/src/core/onCallFirmwareUpdate.ts#L249 
is now duplicated in the code base. it might make sense to move it only here.

Reply:
Do you mean the sanity check for `suspiciously small binary` ?

Reply:
yes